backend/app/datasets/models.py

```diff
@@ -34,6 +33,15 @@ class Dataset(SearchableMixin, db.Model):
 
     tags = db.relationship('Tag', secondary=tags, lazy='subquery', backref=db.backref('datasets', lazy=True))
 
+    # Citation Fields
+    cite_title = db.Column(db.String(1000), nullable=True)
+    cite_authors = db.Column(db.String(1000), nullable=True)
+    cite_venue = db.Column(db.String(1000), nullable=True)
+    cite_year = db.Column(db.Integer, nullable=True)
+
+    # Old Citation Field
+    old_citation = db.Column(db.String(1000), nullable=True)
+
```

Comment:
Should we allow datasets to be uploaded with all null values for citations? I thought we wanted every dataset to require it.

Reply:
I think we ran into some error when we uploaded some manually and had to change it to nullable. The requirement it enforced on the frontend though.

Reply:
The error came from trying to migrate over old datasets that did not have any citation, and therefore gave an error because nullable was set to false. Ideally I think we would like to keep billable = false, we just need another way to handle the old datasets without citation